src/main/webapp/vendor/startbootstrap/index.html

```diff
@@ -54,7 +54,7 @@
 
       <!-- Nav Item - 4 Year Planner -->
       <li class="nav-item">
-        <a class="nav-link" href="index.html">
+        <a class="nav-link" href="planner.html">
```

Comment:
`/planner` for the href should work, right? That's a lot more standard and better looking.

Reply:
that takes you to the servlet page, not actual html page

Reply:
Oh right. It seems off to me but it's fine for now.